index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>YouTube</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <ul>
+                <li><a href="#">___</a></li>
+                <li><a href="#">___</a></li>
+                <li><a href="#">___</a></li>
+            </ul>
+        </nav>
+        <a href="#">YouTube</a>
+        <input id="search" name="search">
+    </header>
+    <div class="main">
+        <div class="video">
+                <iframe width="560" height="315" src="https://www.youtube.com/embed/DEYEmD-NmhI" 
+                    title="YouTube video player" allow="accelerometer; autoplay; 
+                    clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen>
+                </iframe>
+                <h3>Probably the best brasilian side since 1970</h3>
+        </div>  
+    
+        <aside>
+            <h3>The top stars</h3>
+            <article class="card">
+                <h4>Zico</h4>
+                <a target="_blank" href="imagens/zico.jpg.jpg">
+                <img src="imagens/zico.jpg.jpg" alt="zico">
+                </a>
+                <p><em>The Cock</em></p> 
+            </article>
+
+            <article class="card">
+                <h4>Júnior</h4>
+                <a target="_blank" href="imagens/junior.jpg">
+                <img src="imagens/junior.jpg" alt="junior">
+                </a>
+                <p><em>The Maestro</em></p>  
+            </article> 
+
+            <article class="card">
+                <h4>Sócrates</h4>
+                <a target="_blank" href="imagens/socrates.jpg.jpg">
+                <img src="imagens/socrates.jpg.jpg" alt="socrates">
+                </a>
+                <p><em>The Doctor</em></p>
+            </article>
+
+            <article class="card">
+                <h4>Falcão</h4>
+                <a target="_blank" href="imagens/falcao.jpg">
+                <img src="imagens/falcao.jpg" alt="falcao">
+                </a>
+                <p><em>The King of Rome</em></p> 
+            </article> 
+
+            <article class="card">
+                <h4>Éder</h4>
+                <a target="_blank" href="imagens/Eder-2.jpg">
+                <img src="imagens/Eder-2.jpg" alt="Eder">
+                </a> 
+                <p><em>The Bomber</em></p>
+            </article> 
+
+        </aside>
```

Comment:
- [ ] Please avoid sounding like you are threatening the programmer 😅  phrase the comments politely, as suggestions instead of orders.
For example:

This is a polite suggestion: "I think it would be better to use semantic tags"

This reads like an order: "Use semantic tags"

![image](https://user-images.githubusercontent.com/74744787/160412885-d6f9ce9e-a0ea-41ba-aba8-a98af3ea78eb.png)